content/en/docs/v3.6/learning/api_guarantees.md

```diff
@@ -1,75 +1,60 @@
 ---
-title: KV API guarantees
+title: etcd API guarantees
 weight: 2750
-description: KV API guarantees made by etcd
+description: API guarantees made by etcd
 ---
 
-etcd is a consistent and durable key value store with [mini-transaction][txn]
-support. The key value store is exposed through the KV APIs. etcd tries to
-ensure the strongest consistency and durability guarantees for a distributed
-system. This specification enumerates the KV API guarantees made by etcd.
+etcd is a consistent and durable key value store.
+The key value store is exposed through the KV and Watch [gRPC Services].
+etcd ensures the strongest consistency and durability guarantees for a distributed system.
+This specification enumerates the API guarantees made by etcd.
 
-### APIs to consider
+### RPCs to consider
 
-* Read APIs
-    * range
-    * watch
-* Write APIs
-    * put
-    * delete
-* Combination (read-modify-write) APIs
-    * txn
-* Lease APIs
-    * grant
-    * revoke
-    * put (attaching a lease to a key)
+* KV Service
+  * [Range](../api/#range)
+  * [Put](../api/#put)
+  * [Delete Range](../api/#delete-range)
+  * [Transaction](../api/#transaction)
+* Watch Service
+  * [Watch](../api/#watch-api)
+* Lease Service
+  * [Lease grant](../api/#obtaining-leases)
+  * [Keep alive](../api/#keep-alives)
```

Comment:
We are talking about APIs in this doc, why introduce another concept "service"? 
```suggestion
* KV APIs
  * [Range](../api/#range)
  * [Put](../api/#put)
  * [Delete](../api/#delete-range)
  * [Transaction](../api/#transaction)
* Watch APIs
  * [Watch](../api/#watch-api)
* Lease APIs
  * [Grant](../api/#obtaining-leases)
  * [Revoke]
  * [Keep alive](../api/#keep-alives)
  * [Put (attaching a lease to a key)]
```

Reply:
I do not see any confusion with regard to "V2" or "V3 API" here. Let's see other maintainer's opinion.

Reply:
I didn't mean confusion, I mean that we overabused word "API" because we couldn't use specific term like "method"/"service"/"rpc". For me using term "API" for everything muddies the water and hides the real hierarchy. 

There is one "etcd API", that consists of multiple services like "KV service", which exposes multiple rpcs like "Range" or "Put". This is naming is borrowed from grpc which is used in our api documentation https://etcd.io/docs/v3.5/learning/api/.

I'm not attached to grpc naming, but we should pick up a unified naming scheme. For now being consistent with https://etcd.io/docs/v3.5/learning/api/ seemed like a best option.

Reply:
cc @jberkus on the naming convention

Reply:
Applying the suggestion as this discussion is not relevant to goal of this PR.